### PR TITLE
React16

### PR DESCRIPTION
**Pull request by alextrastero**
Initial PR for react 16

### CODE REVIEW

File: test/CarouselSpec.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ describe('<Carousel />', () => {
     const fixedItem = <span>Do you rock!?</span>;
     wrapper = renderer({ images, fixedItem });
     expect(wrapper.find('.carousel-fixed-item').length).to.eq(1);
-    expect(wrapper.find('.carousel-fixed-item').childAt(0).node).to.eq(fixedItem);
+    expect(wrapper.find('.carousel-fixed-item').childAt(0).equals(fixedItem)).to.eq(true);
```
Comment (isaiah) [inline test/CarouselSpec.js]: Why is this change?

**Comment by jbeatty80** [comment]
Would really like to see this get finished and merged. 👍

File: test/CarouselSpec.js (hunk 1 above)
Comment (alextrastero) [reply test/CarouselSpec.js]: This is a breaking change from `enzyme`: https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#private-properties-and-methods-have-been-removed

Comment (jbeatty80) [reply test/CarouselSpec.js]: Good catch, I'll have to keep an eye out for that elsewhere.

**Comment by jroyleaz** [comment]
@isaiah Any movement on this or anything that can be done to help?

**Comment by isaiah** [comment]
@jroyleaz Let's get the party started :1st_place_medal: 
@alextrastero Could you prepare a new release?

**Comment by alextrastero** [comment]
The docs are not working at the moment, but we can get this version out for sure :)

**Comment by jroyleaz** [comment]
Thanks ya'll, it's appreciated!

**Pull request merged**